examples/pointcloud_kdd_radius.cpp

```diff
@@ -66,7 +66,7 @@ void kdtree_demo(const size_t N)
 	// ----------------------------------------------------------------
 	{
 		size_t num_results = 5;
-		std::vector<size_t>   ret_index(num_results);
+		std::vector<uint32_t>   ret_index(num_results);
```

Comment:
I'm puzzled by this change (or rather the change that makes this change necessary). For us it breaks backwards comparibility https://github.com/libMesh/libmesh/pull/3122

Reply:
Aditionally to that, for msvc there are warnings due to conversion loss with size_t when using the default AccessorType = uint32_t value.
`1>C:\dev\GSI\Vstars\packages\nanoflann.1.4.2.14\lib\native\include\nanoflann.hpp(1497,1): warning C4267: '=': conversion from 'size_t' to '_Ty', possible loss of data
1>        with
1>        [
1>            _Ty=std::_Vbase
1>        ]`

For KDTreeSingleIndexAdaptor settings the typename AccessorType = uint32_t to size_t leads to some different conversion errors which seem to imply that internally a uint32_t is being used somewhere.

`1>C:\dev\GSI\Vstars\packages\nanoflann.1.4.2.14\lib\native\include\nanoflann.hpp(1561,1): warning C4267: 'argument': conversion from 'size_t' to 'const AccessorType', possible loss of data
1>        with
1>        [
1>            AccessorType=uint32_t
1>        ]`